src/main/java/org/commcare/cases/instance/CaseInstanceTreeElement.java

```diff
@@ -27,6 +27,10 @@
     private final static XPathPathExpr CASE_INDEX_EXPR = XPathReference.getPathExpr("index/*");
     private final static XPathPathExpr OWNER_ID_EXPR = XPathReference.getPathExpr("@owner_id");
     private final static XPathPathExpr EXTERNAL_ID_EXPR = XPathReference.getPathExpr("@external_id");
+    private final static XPathPathExpr CATEGORY_EXPR = XPathReference.getPathExpr("category");
```

Comment:
We discussed these as being `@` expressions (IE: In the attribute set for the `<case>` node rather than a child element), right? 

Is there a reason you went with this structure?

Reply:
I wanted the same expression to work before and after the change, so projects would be able to make the ordering changes now without having to worry about when this gets deployed, but i do like the idea of `@` for consistency with other indexed properties, so see both sides of this.

Reply:
My intuition is that I don't think it's safe for us to index individual properties, and I also don't' want to set an expectation with project teams that we'll configure request contracts in a way that fits their custom workflows and improve performance in place. 

The property aliasing is already more custom than I'm comfortable with, but we will at least be able to gracefully move away from that. Adding the custom query indexes feels a "forever" change that I'm not ok making part of CommCare's permanent request contract.